big_tests/tests/sm_SUITE.erl

```diff
@@ -159,6 +174,11 @@ register_some_smid_h(Config) ->
     TestSmids = lists:map(fun register_smid/1, lists:seq(1, 3)),
     [{smid_test, TestSmids} | Config].
 
+init_per_testcase(no_crash_if_stream_mgmt_disabled = CN, Config) ->
+    Config2 = dynamic_modules:save_modules(domain(), Config),
```

Comment:
Isn't it a duplicate of init for a group?

Reply:
Indeed, there's no such testcase even. This was my initial test, until I realised there are two different conditions to test and then made it into a group. Removing this trash, good catch!